### PR TITLE
Bug 1813451 - Fix accessibility issues on History deletion screen

### PR DESCRIPTION
**Pull request by gitstart**
The PR fixes accessibility issue on the History deletion screen. The fixes include:
- ~Increased contrast ratio on `Delete` and `Cancel` buttons (achieved by making the card background darker)~
- Dynamic height of radio buttons (achieved with `wrap_content` property)

PS: This is a minor UI change only, and thus does not require writing tests under PR checklist

### Fix description
Updated card background color to achieve the desired contrast ratio.

### Fix screenshot/recording
fix

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1813451

### CODE REVIEW

File: android-components/plugins/dependencies/src/main/java/Gecko.kt [hunk 1]
```diff
@@ -9,7 +9,7 @@ object Gecko {
     /**
      * GeckoView Version.
      */
-    const val version = "112.0.20230217091713"
+    const val version = "112.0.20230216213942"
```
Comment (Mugurell) [inline android-components/plugins/dependencies/src/main/java/Gecko.kt]: This change is probably not needed.

File: fenix/app/src/androidTest/java/org/mozilla/fenix/ui/OnboardingTest.kt [hunk 2]
```diff
@@ -64,11 +64,7 @@ class OnboardingTest {
             )
             verifyToolbarPlacementCard(isBottomChecked = true, isTopChecked = false)
             verifySignInToSyncCard()
-            verifyPrivacyProtectionCard(
-                settings = activityTestRule.activity.getSettings(),
-                isStandardChecked = true,
-                isStrictChecked = false,
-            )
+            verifyPrivacyProtectionCard(isStandardChecked = true, isStrictChecked = false)
```
Comment (Mugurell) [inline fenix/app/src/androidTest/java/org/mozilla/fenix/ui/OnboardingTest.kt:67]: This change regarding `verifyPrivacyProtectionCard` (here and below) is not needed.

File: fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt [hunk 3]
```diff
@@ -153,9 +151,9 @@ class HomeScreenRobot {
         assertItemWithResIdExists(signInButton)
     }
 
-    fun verifyPrivacyProtectionCard(settings: Settings, isStandardChecked: Boolean, isStrictChecked: Boolean) {
+    fun verifyPrivacyProtectionCard(isStandardChecked: Boolean, isStrictChecked: Boolean) {
```
Comment (Mugurell) [inline fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt:156]: Making a note about not reverting adding the new `settings` parameter.

File: fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt [hunk 4]
```diff
@@ -1062,16 +1060,6 @@ private fun sponsoredShortcut(sponsoredShortcutTitle: String) =
 private fun storyByTopicItem(composeTestRule: ComposeTestRule, position: Int) =
     composeTestRule.onNodeWithTag("pocket.categories").onChildAt(position - 1)
 
-private fun privacyProtectionDescription(settings: Settings): UiObject {
```
Comment (Mugurell) [inline fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt:1062]: We probably do want this method.

File: fenix/app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt [hunk 5]
```diff
@@ -515,18 +512,7 @@ fun PoweredByPocketHeader(
...
-                    role = Role.Button
-                    onClick(label = onClickLabel) {
-                        onLearnMoreClicked(
-                            "https://www.mozilla.org/en-US/firefox/pocket/?$POCKET_FEATURE_UTM_KEY_VALUE",
-                        )
-                        false
-                    }
-                },
-            ) {
+            Column {
```
Comment (Mugurell) [inline fenix/app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:515]: Not sure why this was deleted.

File: taskcluster/ci/external-gradle-dependencies/kind.yml [hunk 6]
```diff
@@ -30,6 +30,7 @@ task-defaults:
             - android-components/plugins/dependencies/src/main/java/DependenciesPlugin.kt
             - android-components/build.gradle
             - android-components/gradle/wrapper/gradle-wrapper.properties
+            - focus-android/buildSrc/src/main/java/FocusDependencies.kt
```
Comment (Mugurell) [inline taskcluster/ci/external-gradle-dependencies/kind.yml:33]: We can probably avoid touching this file.

File: fenix/app/src/main/res/values-night/colors.xml [hunk 7]
```diff
@@ -9,7 +9,7 @@
     <!-- Default Screen background, Frontlayer background, App Bar Top, App Bar Bottom, Frontlayer header -->
     <color name="fx_mobile_layer_color_1">@color/photonDarkGrey60</color>
     <!-- Card background, Menu background, Dialog, Banner -->
-    <color name="fx_mobile_layer_color_2">@color/photonDarkGrey30</color>
+    <color name="fx_mobile_layer_color_2">@color/photonDarkGrey70</color>
```
Comment (Mugurell) [inline fenix/app/src/main/res/values-night/colors.xml:12]: Seems like this is the only change proposed to fix the issue?
Given that this is used in many places all throughout the app the change will have to be approved by UX. Will keep you posted with the discussions result.

**Review by Mugurell — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Seems like the current PR would revert many of the recent changes so it needs to be cleaned up.

Changing a color used in many places all throughout the app has to be confirmed by UX. I'll start such a discussion and keep you posted with the decision.

**Comment by gitstart** [comment]
Thank you @Mugurell for reviewing🙏.
Will wait for the feedback from UX and then do the requested changes. 🚀

File: fenix/app/src/main/res/values-night/colors.xml (hunk 7 above)
Comment (gabrielluong) [reply fenix/app/src/main/res/values-night/colors.xml:12]: These values are provided by UX design system. So, this shouldn't be changed externally.

**Comment by lime124** [comment]
agree with GL - this has been sent for consideration to our design systems team.

**Comment by gitstart** [comment]
Hello @Mugurell,
Requested changes pushed.

**Review by Mugurell — APPROVED** [review]
Thank you for helping with this!
The remaining changes do help improve the accessibility on the history deletion dialog.
Tested with various font sized and the radio options looked great 👍
So this looks good to land.

**Comment by Mugurell** [comment]
bors try

**Review by Mugurell — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**